checker/check_stat.ml

```diff
@@ -28,6 +28,10 @@ let pr_impredicative_set env =
   if is_impredicative_set env then str "Theory: Set is impredicative"
   else str "Theory: Set is predicative"
 
+let pr_rewrite_rules env =
+  if rewrite_rules_allowed env then str "Theory: Rewrite rules are allowed (subject reduction might be broken)"
```

Comment:
I was under the impression that every breakage induced by rewrite rules could be explained by a SR failure, what are you thinking about ?

Reply:
It's more about the implication, there are many possible subject reduction failures which don't break consistency (for instance the type theory of Lean).
ie when we read "subject reduction is broken" you mean "subject reduction and most other good properties are broken" but it could also be read as "subject reduction is broken but most other good properties are preserved".

Reply:
It's now more explicit